docs/Makefile

```
@@ -24,6 +24,7 @@ site-common: clean
 	GO111MODULE=on go run cmd/generate_docs.go gen-changelog-md glooe > content/static/content/glooe-changelog.docgen
 	GO111MODULE=on go run cmd/generate_docs.go gen-security-scan-md gloo > content/static/content/gloo-security-scan.docgen
 	GO111MODULE=on go run cmd/generate_docs.go gen-security-scan-md glooe > content/static/content/glooe-security-scan.docgen
+	GO111MODULE=on go run cmd/generate_docs.go get-enterprise-helm-values 1.1 > content/static/content/glooe-values.docgen
```

why pass 1.1? is that unused?

---

Yeah this is left over from testing 
```suggestion
	GO111MODULE=on go run cmd/generate_docs.go get-enterprise-helm-values > content/static/content/glooe-values.docgen
```